site/_posts/2017-05-01-llw-grimoire.html.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Lovecraftian Last Week in ManageIQ"
+author: d-m-u
+date: 2017-05-01
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+You take up the book and look at the bold inscription on its cover, written in a hand which you learned to recognise a long while back. The manuscript is divided into seven sections. Flipping to the section entitled "Log Analysis During Service Provisioning", you make the mistake of reading the section out loud, a mistake which not even the [removal of createfolder event handler][1] nor the [fixing of vendor key in physical servers][2] could remedy.
+
+As you attempt to render by the almost unpronounceable jumble of letters, you hear it. A chilling, grim, singular giggling, that makes you shudder uncontrollably. No description will suffice, save to make you, dear reader, find such words the stuff of cheap imposters or madness. But you must be warned. The air in front of you grows cold and for an instant, you imagine that you can see beyond this frail world. It is a fact that the words you uttered so foolishly had wrought eternal havoc. Whether any could have lived through it is gravely open to doubt. The giggling grows in horrible immensity. You are seized suddenly by gigantic talons attached to a creature so horrible you dare not describe it in any significant detail. Red, dripping, a blob of pulsing jelly, with uncountable tentacled trunks, a monster from the stars which became more clearly defined as it continued to drain you of your blood. And in an instant, it was gone, the ghastly giggling still ringing in your ears as you fall to the floor, whispering, _but I [updated orders to derive action from order services product features][2]!_ You expire after incessant, pleading cries to be saved from some escaped denizens of hell. Even [scopes + vms][3] cannot save you. Death comes unmercifully in a way which cannot be told.
```

Comment:
Is `[2]` pointing where you expected it too? - If it is maybe I missed the context.